### ISSUE TITLE
line-clamp-1 conflicts with `flex` in Select.Trigger when using twmerge

### ISSUE THREAD

**Issue by saturnonearth** [huntabyte/shadcn-svelte#708]
### Describe the bug

In `Select.Trigger`, The `flex` class is being removed by `twMerge` beceause there is also `line-clamp-1` which contains a `display` CSS property.

### Reproduction

Create a basic Select element - using the latest twMerge

### System Info

```bash
System:
    OS: macOS 14.2.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 64.34 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.10.0 - ~/Library/Caches/fnm_multishells/18288_1706395794605/bin/node
    npm: 10.2.3 - ~/Library/Caches/fnm_multishells/18288_1706395794605/bin/npm
    pnpm: 8.13.1 - /opt/homebrew/bin/pnpm
    bun: 1.0.4 - ~/.bun/bin/bun
  Browsers:
    Chrome: 120.0.6099.234
    Edge: 121.0.2277.83
    Safari: 17.2.1
  npmPackages:
    @sveltejs/kit: ^2.5.0 => 2.5.0
    bits-ui: ^0.15.1 => 0.15.1
    cmdk-sv: ^0.0.13 => 0.0.13
    formsnap: ^0.4.3 => 0.4.3
    mode-watcher: ^0.1.2 => 0.1.2
    svelte: 4.2.9 => 4.2.9
    sveltekit-superforms: 2.0.0-alpha.31 => 2.0.0-alpha.31
```

### Severity

annoyance

**Comment by Scooter1337** [comment]
I believe my latest PR solved this issue, please see #705

**Comment by huntabyte** [comment]
Closed by #705
